server/asset/eth/eth_test.go

```diff
@@ -180,7 +180,7 @@ func TestDecodeCoinID(t *testing.T) {
 			t.Fatalf("want flags value of %v but got %v for test %v",
 				test.wantFlags, flags, test.name)
 		}
-		if addr != test.wantAddr {
+		if !bytes.Equal(addr[:], test.wantAddr[:]) {
```

Comment:
comparing slices with bytes.Equal is fine, but I'd probably go with `if *addr != *test.wantAddr` a la `(*chainhash.Hash).IsEqual`

https://github.com/decred/dcrd/blob/48627ee18c3d43319d9b27d51b70afe0e5eb6eb2/chaincfg/chainhash/hash.go#L63-L71

Reply:
I forgot this worked.